.drone.yml

```diff
@@ -21,15 +21,15 @@ trigger:
 
 linting: &linting
   pull: if-not-exists
-  image: node:14
+  image: node:18
```

Comment:
use `node:lts` it'll use node 18 LTS version

Reply:
aah thanks!